app/src/main/res/layout/fragment_review_loan_application.xml

```diff
@@ -103,7 +104,7 @@
                 android:layout_marginLeft="@dimen/default_padding"
                 android:layout_marginStart="@dimen/default_padding"
                 android:layout_width="match_parent"
-                android:text="143.00"
+                tools:text="@string/dummy_principal_amount"
```

Comment:
1 comment, when using tools:text there is no need to use a string resource as its only meant to be used for the layout editor. We can directly put `tools:text=143.00`